cogboard-app/src/main/resources/initData/credentials.json

```diff
@@ -16,6 +16,15 @@
       "user": "mock",
       "label": "SSH Cred",
       "id": "credential4"
+    },
+    {
+      "sshKeyPassphrase": "",
```

Comment:
widzę tylko dodanie credentialsów, a dodanie widżetu do configa początkowego? chodzi o to, żeby taki LogsViewer się nam pokazał domyślnie na którymś boardzie po wystartowaniu apki

Reply:
Domyślny widget musiał być stworzony w innym branchu, zapewne na potrzeby testów. Można go znaleźć w pliku  cogboard/cogboard-app/src/main/resources/initData/config.json po wyszukaniu widget95

Reply:
ok, git